coruscant/settings.py

```
@@ -46,13 +48,40 @@
+                return None
+            for _ in range(3):
+                if frame.f_back is None:
+                    break
+                frame = frame.f_back
+            if frame is None:
+                return None
+            local_vars = {k: v for k, v in frame.f_locals.items() if not k.startswith("__") and not callable(v)}
+            if not local_vars:
+                return None
+            return ", ".join(f"{k}={v!r}" for k, v in local_vars.items())
+        except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)